src/test/java/io/logz/guice/jersey/JettyFilterRegistrationTest.java

```diff
@@ -0,0 +1,50 @@
+package io.logz.guice.jersey;
+
+import io.logz.guice.jersey.configuration.JerseyConfiguration;
+import io.logz.guice.jersey.configuration.JerseyConfigurationBuilder;
+import io.logz.guice.jersey.filters.JettyTestFilter;
+import io.logz.guice.jersey.resources.TestResource;
+import io.logz.guice.jersey.supplier.JerseyServerSupplier;
+import me.alexpanov.net.FreePortFinder;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.servlet.FilterHolder;
+import org.glassfish.jersey.server.ResourceConfig;
+import org.junit.Test;
+
+import javax.servlet.DispatcherType;
+import javax.ws.rs.core.Response;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class JettyFilterRegistrationTest {
+
+    @Test
+    public void testJettyFilter() throws Exception {
+        ResourceConfig resourceConfig = new ResourceConfig().registerClasses(TestResource.class);
+        JerseyConfigurationBuilder configurationBuilder = JerseyConfiguration.builder()
+                .withResourceConfig(resourceConfig);
+        int port = FreePortFinder.findFreeLocalPort();
```

Comment:
Why? You have in `JettyServerSupplier` no?

Reply:
In the `JettyServerSupplier` I only added the web app context configuration to the last constructor. If you'd like I can add this everywhere else

Reply:
Before your addition and even after, JettyServerSupplier has this
```
        int port = FreePortFinder.findFreeLocalPort();
        configurationBuilder.addPort(port);
```

Why do you need to have to come up with a free port from outside, in the test method it self, in addition to this?